requirements.txt

```diff
@@ -17,4 +17,5 @@ pyccl
 sacc
 fgspectra>=1.1.0
 syslibrary
+fast-pt
```

Comment:
@itrharrison do we want to have fast-pt (and velocileptors below) as default requirements when installing soliket?